### PR TITLE
Allow absolute paths to be passed to requireReporter r=lightsofapollo

### PR DESCRIPTION
**Pull request by gaye**

### CODE REVIEW

File: bin/marionette-mocha [hunk 1]
```diff
@@ -10,6 +10,22 @@ var requireResolve = require('../lib/require_resolve');
 
 var OPTS_FILE = 'marionette-mocha.opts';
 
+var argv = ['test/', 'tests/', '.'].reduce(function(argv, path) {
```
Comment (gaye) [inline bin/marionette-mocha]: Just moved this up so that we can bind `argv` to `requireReporter`

File: lib/require_reporter.js [hunk 2]
```diff
@@ -0,0 +1,20 @@
+/**
+Helper to import the mocha reporters...
+*/
+module.exports = function requireReporter(argv, path) {
+  var reporter;
+  [
+    argv.reporter,
+    path,
```
Comment (gaye) [inline lib/require_reporter.js]: Should also try absolute path

File: lib/require_resolve.js [hunk 3]
```diff
@@ -1,6 +1,6 @@
 var fsPath = require('path');
 
-module.exports = function requireResovle(path) {
+module.exports = function requireResolve(path) {
```
Comment (gaye) [inline lib/require_resolve.js]: lol

**Comment by lightsofapollo** [comment]
@gaye can we add a test?

**Pull request merged**